jobbergate-cli/jobbergate_cli/config.py

```diff
@@ -84,7 +83,8 @@ def compute_extra_settings(cls, values):
 
         values["JOBBERGATE_CLUSTER_LIST_PATH"] = cache_dir / "clusters.json"
 
-        values.setdefault("OIDC_LOGIN_DOMAIN", values["OIDC_DOMAIN"])
+        if "OIDC_LOGIN_DOMAIN" not in values or not values.get("OIDC_LOGIN_DOMAIN", None):
+            values["OIDC_LOGIN_DOMAIN"] = values["OIDC_DOMAIN"]
```

Comment:
can we have just `OIDC_DOMAIN`? `OIDC_LOGIN_DOMAIN` doesn't make sense at all, it should have only one of them

Reply:
Sure, working on it.

Reply:
solved in [e9e4be5](https://github.com/omnivector-solutions/jobbergate/pull/125/commits/e9e4be54ed4e42f36b295a779594c006488e0777)